cterasdk/core/logs.py

```diff
@@ -11,28 +11,70 @@ class Logs(BaseCommand):
     Portal Logs APIs
     """
 
-    def get(self, topic=enum.LogTopic.System, minSeverity=enum.Severity.INFO, originType=enum.OriginType.Portal, before=None, after=None):
+    def device(self,
+               name,
+               topic=enum.LogTopic.System,
+               min_severity=enum.Severity.INFO,
+               before=None,
+               after=None,
+               filters=None):
+        """
+        Get device logs from the Portal
+
+        :param str name: Name of a device
+        :param cterasdk.core.enum.LogTopic,optional topic: Log topic to get, defaults to cterasdk.core.enum.LogTopic.System
+        :param cterasdk.core.enum.Severity,optional min_severity:
+         Minimun severity of logs to get, defaults to cterasdk.core.enum.Severity.INFO
+        :param str,optional before: Get logs before this date (in format "%m/%d/%Y %H:%M:%S"), defaults to None
+        :param str,optional after: Get logs after this date (in format "%m/%d/%Y %H:%M:%S"), defaults to None
+        :param list[cterasdk.core.query.FilterBuilder],optional filters: List of additional filters, defaults to None
+
+        :return: Iterator for all matching logs
+        :rtype: cterasdk.lib.iterator.Iterator[cterasdk.object.Object]
+        """
+        return self.logs(topic, min_severity, enum.OriginType.Device, name, before, after, filters)
+
+    def logs(self,
```

Comment:
Why do we need the name change here?

Reply:
ah i don't like the inconsistency we had with camel case vs the pythonic way of naming fields. Old habits from coding java

Reply:
Oh, I meant the method name get->logs (I got the parameter names)

Reply:
Not for any good reason. I just thought it's more aligned with core/devices. But I just changed it back. See update

Reply:
@ygalblum - I changed it back, lmk if you approve